frameworks/PHP/workerman/benchmark_config.json

```diff
@@ -0,0 +1,29 @@
+{
+  "framework": "php",
```

Comment:
@kenny1har Can you change this to "workerman"

Reply:
changed.